src/controllers/ManageTermController.java

```diff
@@ -3,33 +3,42 @@
 import java.net.URL;
 import java.util.ResourceBundle;
 
+import database.TermDatabaseController;
 import javafx.fxml.FXML;
 import javafx.fxml.Initializable;
 import javafx.scene.control.DatePicker;
+import javafx.scene.control.TextField;
 import javafx.stage.Stage;
 
 public class ManageTermController implements Initializable {
-	@FXML
-	DatePicker startDatePicker;
-	@FXML
-	DatePicker endDatePicker;
+	@FXML DatePicker startDatePicker;
+	@FXML DatePicker endDatePicker;
+	@FXML TextField txtFldDescription;
```

Comment:
😠

Reply:
Should I put the annotations above the variable declaration master? :disappointed:

Reply:
yes 🗡

Reply:
Your will, my hands.

Reply:
🖤